strapi/strapi
# fix: change status on locale picker

### What does it do?

Change LocalePicker to match style and give user more information about creating an entry in a new locale.

![Screenshot 2024-09-09 at 18 07 59](https://github.com/user-attachments/assets/fc925756-dc74-4b34-bbd9-5396cd7afeb8)

### How to test it?

(Try for both: content types with d&p) 

1. Enter a entry with i18n enabled.
2. Try to create, publish, etc, same entry in multiple locales and see results in the locale picker.

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/19960
